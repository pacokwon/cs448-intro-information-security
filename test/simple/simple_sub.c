#include <stdio.h>
#include <stdlib.h>

int main()
{
    char input[65536];
    int x = 7;
    int y;
    fgets(input, sizeof(input), stdin);

    y = x - atoi(input);

    int z = 20 / y; // Error
}
