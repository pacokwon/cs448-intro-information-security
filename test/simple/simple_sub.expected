Divide-by-zero detected at line 6 and col 16
